examples/dns/dnsmasq.conf

```diff
@@ -0,0 +1,3 @@
+log-queries
+server=127.0.0.11
```

Comment:
QUES: Is the IP typo? I actually do not quite understand what this flag means. In the doc it says `Specify IP address of upstream servers directly`

Reply:
Not a typo (I can add a comment). 

Docker's internal embedded DNS happens to be at `127.0.0.11`, so by setting this as an upstream server we are telling dnsmasq to relay requests to the Docker's DNS for names which dnsmasq can't resolve.